.github/dependabot.yml

```diff
@@ -22,3 +22,13 @@ updates:
       firecracker:
         patterns:
           - "*"
+        exclude-patterns:
+          - "vmm-sys-util"
+          - "kvm-bindings"
+          - "kvm-ioctls"
+          - "vm-memory"
+          - "vhost"
+          - "linux-loader"
+          - "vm-allocator"
+          - "event-manager"
+          - "vm-superio"
```

Comment:
Do we have get separate PRs for those, or do we need to do it manually?

Reply:
We're supposed to get one separate PR that combines all of these dependencies. There's two groups in the config, and this exclude applies to the second, so the "normal" PR will exclude the rust-vmm dependencies, but the "rust-vmm" PR will include then (and only them, because that group has a whitelist).

According to the documentation, this exclude shouldn't be needed because the rust-vmm group is listed first, but it seems that because we currently have a firecracker PR from before the configuration change (e.g. with an update for vmm-sys-util), dependabot thinks that "well there's nothing to do for the rust-vmm group because vmm-sys-util is already in this other PR", so this is my way of hopefully telling it to stop doing that haha